src/rabbit_mgmt_gc.erl

```diff
@@ -235,3 +235,18 @@ gc_entities(Table, QueueGbSet, ExchangeGbSet) ->
                       gc_object(Q, Table, Object, QueueGbSet),
                       gc_entity(X, Table, X, ExchangeGbSet)
               end, none, Table).
+
+is_process_alive_in_cluster(Pid) ->
```

Comment:
I believe we can use https://github.com/rabbitmq/rabbitmq-common/blob/master/src/rabbit_misc.erl#L864 here.